torch/csrc/jit/tensorexpr/types.h

```diff
@@ -31,6 +31,42 @@ enum class ScalarType : int8_t {
   NumOptions
 };
 
+static inline const char* toString(ScalarType t) {
```

Comment:
There already exists a function of the definition `std::string to_string(const ScalarType& type);`. Is this different enough to need another version? What is the distinction and can we make it clearer in the name/comments?